compose/build.gradle.kts

```diff
@@ -78,6 +78,7 @@ val devConfigurations: List<Configuration> = listOf(internalDebugImplementation,
 
 dependencies {
 
+    implementation(project(mapOf("path" to ":domain")))
```

Comment:
@VanyaHuamanBottleRocket @BottleRocket-Colin Will adding this dependency on `domain` to the `compose` module slow down time to render for the compose previews? If so, we should consider another approach

Reply:
The preview time seemed ok to me. @BottleRocket-Colin What do you think?

Reply:
@br-jared-stockton , @VanyaHuamanBottleRocket  - It shouldn't; being a separate module should keep changes and needed rebuilds separated.